src/main/java/ludogorie_soft/reservations_platform_api/dto/BookingResponseWithCustomerDataDto.java

```diff
@@ -0,0 +1,25 @@
+package ludogorie_soft.reservations_platform_api.dto;
+
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Data
+public class BookingResponseWithCustomerDataDto {
+
+    private UUID id;
```

Comment:
Instead of repeating all properties can we make this DTO class to contain just 2 other DTOs: BookingRequestDto and BookingRequestCustomerDataDto. And we will be able to get all the necessary properties from there.

Reply:
If we return the 2 DTOs, we will be returning propertyId, which we currently do not, and not returning the totalPrice, which we currently do and can be added as a separate field. Do we want to return the propertyId?

Reply:
changed